devices.js

```diff
@@ -2014,6 +2014,30 @@ const devices = [
             await configureReporting.batteryVoltage(endpoint1);
         },
     },
+    {
+        zigbeeModel: ['Lightify Switch Mini blue'],
```

Comment:
As this is identical to the `AC0251100NJ` I think we should merge it with that one, can you change https://github.com/Koenkk/zigbee-herdsman-converters/pull/768/files#diff-6c9a6acf22f90d1c6e524d9f3c5c1745R1994 to zigbeeModel: ['Lightify Switch Mini', 'Lightify Switch Mini blue'],

Reply:
Done 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/768/commits/cf2ae906cf9de4e00c2757d0ff7c21a7633f9fbb#diff-6c9a6acf22f90d1c6e524d9f3c5c1745R1994